configure.sh

```diff
@@ -36,6 +36,7 @@ function write_action_env_to_bazelrc() {
 
 [[ -f .bazelrc ]] && rm .bazelrc
 pip install $QUIET_FLAG -r requirements.txt
+pip3 install $QUIET_FLAG -r requirements.txt
```

Comment:
We run ./configure from inside a virtualenv with either py3 or py2 as default, meaning the line on 38 will work appropriately, and this line should in theory work, if redundantly, but also could theoretically result in some strange interactions. Maybe this switching should be handled on the user side with virtualenvs rather than in the script itself? Ie, we provide instructions on setting up a py2 and py3 virtual env with TF (or rather, link to the same in the TF docs). That's closer to best practice for TF in any case.

Reply:
Sounds good! I will see would i can do

Reply:
Let's remove pip3, Moritz ?